manifests/instance.pp

```diff
@@ -35,6 +38,8 @@
   $java_home              = undef,
   $use_jsvc               = undef,
   $use_init               = undef,
+  $copy_from_home         = undef,
+  $copy_from_home_mode    = undef,
```

Comment:
Don't you need an extra variable that holds the files that you want to control/copy? There are scenarios where you don't want to copy all the default files.